packages/reporter/src/test/test.tsx

```diff
@@ -182,7 +183,7 @@ class Test extends Component<TestProps> {
   _controls () {
     let controls: Array<JSX.Element> = []
 
-    if (this.props.model.state) {
+    if (this.props.model.state === 'failed') {
```

Comment:
This is a drive-by fix. Right now we render this control even when the test hasn't failed